test/scripts/e2e_subs/tealprogs/app-escrow.teal

```diff
@@ -126,17 +126,17 @@ not_deposit:
         -
         app_local_put
 
-        tx_begin
+        itxn_begin
          int pay
-         tx_field TypeEnum
+         itxn_field TypeEnum
 
          txn ApplicationArgs 1
          btoi
-         tx_field Amount
+         itxn_field Amount
 
          txn Sender
-         tx_field Receiver
-        tx_submit
+         itxn_field Receiver
+        itxn_submit
```

Comment:
not sure if it fits but consider adding new itxn opcode into e2e test

Reply:
I do one in the integration test (not true e2e).  I will add a e2e sub test while in betanet (after making it so I can use python!)

Reply:
yes, these e2e_subs tests are shared with the indexer so it is worth having new opcodes/side effects here